projects/ccf-eui/src/app/shared/components/spatial-search-config/spatial-search-config.component.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  EventEmitter,
+  HostBinding,
+  Input,
+  Output,
+} from '@angular/core';
+import { MatDialogRef } from '@angular/material/dialog';
+import { ALL_POSSIBLE_ORGANS, OrganInfo } from 'ccf-shared';
+
+
+export type Sex = 'male' | 'female';
+
+@Component({
+  selector: 'ccf-spatial-search-config',
+  templateUrl: './spatial-search-config.component.html',
+  styleUrls: ['./spatial-search-config.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class SpatialSearchConfigComponent {
+  @HostBinding('class') readonly className = 'ccf-spatial-search-config';
+
+  @Input() allOrgans: OrganInfo[] = ALL_POSSIBLE_ORGANS;
+
+  @Input() selectedOrgan?: OrganInfo;
+
+  @Output() readonly sexChange = new EventEmitter<Sex>();
+
+  @Output() readonly organChange = new EventEmitter<OrganInfo>();
+
+  @Output() readonly itemSelected = new EventEmitter<{ sex: Sex; organ: OrganInfo | undefined }>();
+
+  filteredOrgans: OrganInfo[];
```

Comment:
Needs to be updated when `organs` inputs changes. Use ngOnChanges to detect and update